modules/monitoring-configuring-alert-routing-default-platform-alerts.adoc

```diff
@@ -141,22 +152,30 @@ route:
     - "alertname=Watchdog"
     repeat_interval: 2m
     receiver: watchdog
-  - matchers: # <1>
-    - "<your_matching_rules>" # <2>
-    receiver: <receiver> # <3>
+  - matchers:
+    - "<your_matching_rules>"
+    receiver: <receiver>
 # ...
 ----
-<1> Use the `matchers` key name to specify the matching rules that an alert has to fulfill to match the node.
-If you define inhibition rules, use `target_matchers` key name for target matchers and `source_matchers` key name for source matchers.
-<2> Specify labels to match your alerts.
-<3> Specify the name of the receiver to use for the alerts.
++
+--
+where:
+
+`matchers`::
```

Comment:
So `matchers` is a YAML field and `<receiver>` & `<your_matching_rules>` are user replaced values. The question here is should we divide the styles accordingly? :) I will leave it upto you to decide.